src/snowflake/cli/api/project/definition_conversion.py

```diff
@@ -216,17 +216,22 @@ def _find_manifest():
         # glob patterns. The simplest solution is to bundle the app and find the
         # manifest file from the resultant BundleMap, since the bundle process ensures
         # that only a single source path can map to the corresponding destination path
-        try:
-            bundle_map = build_bundle(
-                project_root, Path(native_app.deploy_root), native_app.artifacts
-            )
```

Comment:
switched from `build_bundle` to just `BundleMap` since we don't actually need to copy the artifacts to find the manifest, we just need to resolve the mapping between source and destination